build-tests/heft-node-basic-test/tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "experimentalDecorators": true,
     "strictNullChecks": true,
     "noUnusedLocals": true,
-    "types": ["jest", "node"],
+    "types": ["heft-jest", "node"],
+    "typeRoots": ["node_modules/@types", "node_modules/@rushstack/heft-jest/types"],
```

Comment:
As far as I can tell, this `typeRoots` line is unavoidable unless we publish a `@types/heft-jest` package to be hosted by DefinitelyTyped.  I'm thinking about pursuing that option.